coforma/swift-tech-challenge
# Updated data ingestions

# Updated data ingestion

# Summary
Updates the data ingestion flow to use AI for generative content
Closes #70 
Closes #20 

## Type of change
 New feature (non-breaking change which adds functionality)

# How To Test
1. Drop a file into S3 for instutions -- watch it process. Recommend using one that is only a line or two to verify, otherwise long waits.
2. Drop a file into S3 for applications -- watch it process. Recommend using one that is only a line or two to verify, otherwise long waits.
3. Verify in Dynamo

# Checklist:
